libs/blocks/merch/merch.js

```diff
@@ -120,7 +120,7 @@ window.tacocat.loadPromise = new Promise((resolve) => {
       resolve(false);
     })
     .catch((error) => {
-      console.error('Failed to load tacocat', error);
```

Comment:
window.lana is sure to be defined at this point, right?

Reply:
Was commenting on this too 😄 I see this is used across a lot of files, but I'd argue `window.lana?.log` might be a bit safer

Reply:
Yes `window.lana` should be defined.  It's defined before `loadArea` fires for the document for every consumer:
https://github.com/adobecom/bacom/blob/main/scripts/scripts.js#L181

However it's possible a consumer forgets to add or removes `loadLana` so we should add the `?`